app/src/views/oppsummering/Oppsummering.tsx

```diff
@@ -261,18 +260,23 @@ export const Oppsummering = () => {
                 <FormSummary.Value>
                   <FormSummary.Answers>
                     {inntektsmeldingSkjemaState.naturalytelserSomMistes.map(
-                      (naturalytelse) => (
-                        <FormSummary.Answer key={naturalytelse.navn}>
-                          <FormSummary.Label>
-                            {formatYtelsesnavn(naturalytelse.navn, true)}
-                          </FormSummary.Label>
-                          <FormSummary.Value>
-                            Verdi {formatKroner(naturalytelse.beløp)} (fra og
-                            med{" "}
-                            {formatDatoKort(new Date(naturalytelse.fraOgMed))})
-                          </FormSummary.Value>
-                        </FormSummary.Answer>
-                      ),
+                      (naturalytelse) => {
+                        const fraOgMedStreng = `fra og med ${formatDatoKort(new Date(naturalytelse.fraOgMed))}`;
+                        const tilOgMedStreng = naturalytelse.tilOgMed
+                          ? `til og med ${formatDatoKort(new Date(naturalytelse.tilOgMed))}`
+                          : "";
+
+                        return (
+                          <FormSummary.Answer key={naturalytelse.navn}>
+                            <FormSummary.Label>
+                              {formatYtelsesnavn(naturalytelse.navn, true)}
+                            </FormSummary.Label>
+                            <FormSummary.Value>
+                              {`Verdi ${formatKroner(naturalytelse.beløp)} (${[fraOgMedStreng, tilOgMedStreng].filter(Boolean).join(", ")}) `}
```

Comment:
Litt i hui og hast, but works 🤷

Reply:
<img src="https://media1.giphy.com/media/nZymwrrw9PjDa/giphy.gif"/>

Altså, clever løst det der. 🏇🏻  Men hele denne verdi, fom og tom strengen kunne blitt abstrahert ut i en `formatVerdiOgDatoer` metode eller noe? Enklere å scanne gjennom koden og forstå hva som skjer liksom

Reply:
Yes, enig. Ser ikke for meg at det skal gjenbrukes noe annet sted (altså ikke som en util), bare funksjon i denne fila tenker jeg